RecoParticleFlow/PFProducer/plugins/importers/ImprovedTracksImporter.cc

```diff
@@ -0,0 +1,259 @@
+#include "RecoParticleFlow/PFProducer/interface/BlockElementImporterBase.h"
+#include "DataFormats/ParticleFlowReco/interface/PFBlockElementTrack.h"
+#include "DataFormats/ParticleFlowReco/interface/PFRecTrackFwd.h"
+#include "DataFormats/ParticleFlowReco/interface/PFRecTrack.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/MuonReco/interface/MuonFwd.h"
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "RecoParticleFlow/PFProducer/interface/PFMuonAlgo.h"
+
+class ImprovedTracksImporter : public BlockElementImporterBase {
```

Comment:
"Improved" is generally not a good name
What will happen to the next developments? "ImprovedImproved"?